### ISSUE TITLE
mixxx: build broken

### ISSUE THREAD

**Issue by ebfe** [voidlinux/void-packages#6295]
```
In file included from src/util/math.h:11:0,                                                    
                 from src/util/types.h:4,                                                                        
                 from src/effects/native/reverbeffect.h:10,                                                      
                 from src/effects/native/reverbeffect.cpp:3:                                                     
src/util/fpclassify.h:22:20: error: 'std::util_fpclassify' has not been declared                                 
 #define fpclassify util_fpclassify                                                                              
                    ^                                                                                            
src/util/fpclassify.h:23:18: error: 'std::util_isfinite' has not been declared                                   
 #define isfinite util_isfinite                                                                                  
                  ^                                                                                              
src/util/fpclassify.h:20:15: error: 'std::util_isinf' has not been declared                                      
 #define isinf util_isinf                                                                                        
               ^                                                                                                 
src/util/fpclassify.h:19:15: error: 'std::util_isnan' has not been declared        
 #define isnan util_isnan                                                                                        
               ^                                                                                                 
src/util/fpclassify.h:21:18: error: 'std::util_isnormal' has not been declared                                   
 #define isnormal util_isnormal                                                                                  
                  ^                                                                                              
scons: *** [lin64_build/effects/native/reverbeffect.o] Error 1                                                   
In file included from src/util/math.h:11:0,                                                                      
                 from src/util/types.h:4,                                                                        
                 from src/effects/effectprocessor.h:8,                                                           
                 from src/effects/effectinstantiator.h:7,                                                        
                 from src/effects/effect.h:10,                                                                   
                 from src/effects/effectsbackend.h:9,                                                            
                 from src/effects/native/nativebackend.h:4,                
                 from src/effects/native/nativebackend.cpp:3:                                                    
src/util/fpclassify.h:22:20: error: 'std::util_fpclassify' has not been declared                                 
 #define fpclassify util_fpclassify                                                                              
                    ^                                                                                            
src/util/fpclassify.h:23:18: error: 'std::util_isfinite' has not been declared                                   
 #define isfinite util_isfinite                                                                                  
                  ^                                                                                              
src/util/fpclassify.h:20:15: error: 'std::util_isinf' has not been declared                                      
 #define isinf util_isinf                                                                                        
               ^                                                                                                 
src/util/fpclassify.h:19:15: error: 'std::util_isnan' has not been declared                                      
 #define isnan util_isnan                                                                                        
               ^                                                                                                 
src/util/fpclassify.h:21:18: error: 'std::util_isnormal' has not been declared                                   
 #define isnormal util_isnormal                                                                                  
                  ^                                                                                              
scons: *** [lin64_build/effects/native/nativebackend.o] Error 1                                                  
scons: building terminated because of errors.                                                                    
=> ERROR: mixxx-2.0.0_3: failed to run do_build() at line 32.   
```

**Comment by Gottox** [comment]
Related debian bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=831071

**Comment by Gottox** [comment]
fixed in 2e97310ae8f905fd8391a246e12a5f277d9b020c
